### PR TITLE
Design Attractive footer

### PR DESCRIPTION
**Pull request by Neutx**
## Purpose

The purpose of this Pull Request is to fix #<194>

## Goals

Add an attractive footer

### Screenshots

https://cdn.discordapp.com/attachments/802898705002266644/1060562215834497034/image.png
### Preview Link

https://pr-206-new-yoshitha-website.surge.sh/

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Can you change the title of this PR @Neutx

File: assets/css/styles.css [hunk 1]
```diff
@@ -343,3 +515,6 @@
     top: 0px;
   }
 }
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:518]: Can you remove these unwanted lines @Neutx

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 1 above)
Comment (YoshithaRathnayake) [reply assets/css/styles.css:518]:
```suggestion
```

File: assets/css/styles.css [hunk 2]
```diff
@@ -343,3 +515,6 @@
     top: 0px;
   }
 }
+
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:519]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css [hunk 3]
```diff
@@ -343,3 +515,6 @@
     top: 0px;
   }
 }
+
+
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:520]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css [hunk 4]
```diff
@@ -1,3 +1,175 @@
...
+  .social-icons li.title
+  {
+    display:block;
+    margin-right:0;
+    font-weight:600
+  }
+}
+
+/*End Footer*/
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:167]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: assets/css/styles.css [hunk 5]
```diff
@@ -1,3 +1,175 @@
...
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:168]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: assets/css/styles.css [hunk 6]
```diff
@@ -1,3 +1,175 @@
...
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:169]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: assets/css/styles.css [hunk 7]
```diff
@@ -1,3 +1,175 @@
...
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:170]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: assets/css/styles.css [hunk 8]
```diff
@@ -1,3 +1,175 @@
...
+
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:171]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 9]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 10]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:2]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 11]
```diff
@@ -1115,3 +1117,144 @@ html {
   font-size: 1.1rem;
   transition: 0.2 ease;
 }
+
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1120]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 12]
```diff
@@ -1115,3 +1117,144 @@ html {
   font-size: 1.1rem;
   transition: 0.2 ease;
 }
+
+
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1121]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 13]
```diff
@@ -1115,3 +1117,144 @@ html {
...
+  color: #919191;
+}
+
+/*End Footer*/
+.a-CTA {
+  border-bottom: 2px solid #a96700;
+  padding-bottom: 4px;
+  letter-spacing: 1.5px;
+  font-size: 18px;
+}
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1260]: Add end of the line @Neutx 
```suggestion
}

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 14]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+  <i class="fas fa-angle-double-up arrow-up"></i>
+  <ul class="info">
+      <li>&copy; Yoshitha Rathnayake</li>
+      <li>Kandy, Sri Lanka</li>
+      <li>Mob: +91 XXXXXXXXXX</li>
+      <li>Made with love by <a href="#">Yoshitha</a> </li>
+  </ul>
+</div>
+<!--End Copy-Right-->
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:265]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: dark/index.html [hunk 15]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:266]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: dark/index.html [hunk 16]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+      <li>&copy; Yoshitha Rathnayake</li>
+      <li>Kandy, Sri Lanka</li>
+      <li>Mob: +91 XXXXXXXXXX</li>
+      <li>Made with love by <a href="#">Yoshitha</a> </li>
+  </ul>
+</div>
+<!--End Copy-Right-->
+
+  
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:267]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: dark/index.html [hunk 17]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:268]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 18]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+
   </section>
 
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:272]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: dark/index.html [hunk 19]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:273]: Can you remove these unwanted lines @Neutx 
```suggestion
```

File: dark/index.html [hunk 20]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+  
+
+
+
   </section>
 
+
+  
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:274]: Can you remove these unwanted lines @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 21]
```diff
@@ -210,18 +210,69 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+</body>
```
Comment (YoshithaRathnayake) [inline dark/index.html:275]: Remove this unwanted body ending tag @Neutx 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 22]
```diff
@@ -244,47 +295,5 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
 
   <!-- Internal JavaScript -->
   
-  <script>
```
Comment (YoshithaRathnayake) [inline dark/index.html:247]: Don't remove this script content, undo that change @Neutx

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 23]
```diff
@@ -244,47 +295,5 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
-    
-    const menu_toggle = document.querySelector(".menu-toggle");
-    const sidebar = document.querySelector(".sidebar");
-
-    menu_toggle.addEventListener("click", () => {
-      menu_toggle.classList.toggle("is-active");
-      sidebar.classList.toggle("is-active");
-    });
-  </script>
-</body>
```
Comment (YoshithaRathnayake) [inline dark/index.html:288]: Don't remove this body ending tag, keep them like earlier @Neutx

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — COMMENTED** [review]
@Neutx Sorry, I can't accept this PR because there are lot of changes you have done and some of them have changed the main content also, so the footer is very nice if you can undo that changes I can approve this

**Comment by Neutx** [comment]
> Can you change the title of this PR @Neutx

File: assets/css/styles.css (hunk 1 above)
Comment (Neutx) [reply assets/css/styles.css:518]: Lines have been removed

File: assets/css/styles.css (hunk 2 above)
Comment (Neutx) [reply assets/css/styles.css:519]: Line has been removed

File: dark/index.html (hunk 22 above)
Comment (Neutx) [reply dark/index.html:247]: Added it back, it was removed by mistake. Thanks for letting me know.

File: dark/index.html (hunk 23 above)
Comment (Neutx) [reply dark/index.html:288]: Again thanks for reminding the body ending tag was also misplaced.

File: assets/css/styles.css [hunk 24]
```diff
@@ -342,4 +507,4 @@
   .bar-preloader {
     top: 0px;
   }
-}
+}
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:510]: Add a new line at the bottom of this file @Neutx

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 23 above)
Comment (YoshithaRathnayake) [reply dark/index.html]: Thanks for solving them @Neutx

File: dark/index.html (hunk 21 above)
Comment (YoshithaRathnayake) [reply dark/index.html]: No I have inserted an body ending tag after the script tag you have removed it

File: assets/css/styles.css (hunk 24 above)
Comment (Neutx) [reply assets/css/styles.css]: check now

**Review by YoshithaRathnayake — COMMENTED** [review]
Looks good for me @Neutx

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Can you change the preview link to this? @Neutx 
https://pr-206-NEW-YOSHITHA-WEBSITE.surge.sh

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Change the issue number to #194 @Neutx

**Comment by Neutx** [comment]
> Change the issue number to #194 @Neutx

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Change the issue number to this
The purpose of this Pull Request is to fix #194

image

**Review by YoshithaRathnayake — COMMENTED** [review]
Before
image


After
image

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Before
image


After
image

**Pull request closed without merging**